pkg/volume/csi/csi_block.go

```diff
@@ -201,8 +189,8 @@ func (m *csiBlockMapper) MapDevice(devicePath, globalMapPath, volumeMapPath, vol
 	defer cancel()
 
 	globalMapPathBlockFile := devicePath
-	dir, _ := m.GetPodDeviceMapPath()
-	targetBlockFilePath := filepath.Join(dir, "file")
+	dir, file := m.GetPodDeviceMapPath()
```

Comment:
Again returning `"file"` will not work properly for drivers that needs volumeID during Teardown.

Reply:
I will rename ```file``` to ```volumeID``` when ```GetPodDeviceMapPath``` is changed to return ```volumeID```.